packages/pass-style/test/test-far-class-instances.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable max-classes-per-file */
+import { test } from './prepare-test-env-ava.js';
+
+// TODO enable import of getMethodNames without deep import
+// eslint-disable-next-line import/order
+import { getMethodNames } from '@endo/eventual-send/src/local.js';
+import { passStyleOf } from '../src/passStyleOf.js';
+import { FarBaseClass } from '../src/far-class-instances.js';
+
+class FarSubclass1 extends FarBaseClass {
+  double(x) {
+    return x + x;
+  }
+}
+
+class FarSubclass2 extends FarSubclass1 {
+  #y = 0;
+
+  constructor(y) {
+    super();
+    this.#y = y;
+  }
+
+  doubleAdd(x) {
+    return this.double(x) + this.#y;
+  }
+}
+
+test('far class instances', t => {
+  const fb = new FarBaseClass();
+  t.is(passStyleOf(fb), 'remotable');
+  t.deepEqual(getMethodNames(fb), ['constructor']);
+
+  t.assert(new fb.constructor() instanceof FarBaseClass);
+  t.throws(() => fb.constructor(), {
+    // TODO message depends on JS engine, and so is a fragile golden test
```

Comment:
how can this TODO be solved?

isn't it fair to assume all of Endo's tests run in whatever its CI matrix covers?

Reply:
Attn @kriskowal @warner 

To truly not rely on any unspecified JS, all I may test is the error type. But I do like ensuring that the test case is failing for the reason it should. I don't know how to do that without this fragile dependency.

Recommendations?